libs/farcaster-provider/src/lib/FarcasterMonitor.ts

```diff
@@ -185,6 +186,7 @@ function isValidUser(
     fid: 0,
     fname: '',
     withFcastMeLink: true,
+    withHashTagOnly: false,
```

Comment:
This will cause current users that don't have this field to be treated as invalid users. Can we remove this?